test/unit/serialization_sb3.js

```diff
@@ -4,7 +4,7 @@ const VirtualMachine = require('../../src/index');
 const Runtime = require('../../src/engine/runtime');
 const sb3 = require('../../src/serialization/sb3');
 const readFileToBuffer = require('../fixtures/readProjectFile').readFileToBuffer;
-const exampleProjectPath = path.resolve(__dirname, '../fixtures/clone-cleanup.sb2');
+const exampleProjectPath = path.resolve(__dirname, '../fixtures/complex.sb2');
```

Comment:
👀

Reply:
I wanted to keep this as an SB2, so I swapped it when `clone-cleanup` turned into an SB3. I'll undo this change when I make `clone-cleanup` an SB2 again.